test/access/AccessControl.ts

```diff
@@ -103,4 +105,57 @@ describe('AccessControl', function () {
         .withArgs(ROLE, DEFAULT_ADMIN_ROLE, newAdminRole);
     });
   });
+
+  describe('#_getRoleMember(bytes32,uint256)', function () {
+    it('returns the correct member', async function () {
+      await instance.connect(admin).grantRole(ROLE, nonAdmin.address);
+      await instance.connect(admin).grantRole(ROLE, nonAdmin2.address);
+      await instance.connect(admin).grantRole(ROLE, nonAdmin3.address);
+
+      expect(await instance.getRoleMember(ROLE, 0)).to.equal(nonAdmin.address);
+      expect(await instance.getRoleMember(ROLE, 1)).to.equal(nonAdmin2.address);
+      expect(await instance.getRoleMember(ROLE, 2)).to.equal(nonAdmin3.address);
+    });
+
+    describe('reverts if', function () {
+      it('role does not exist', async function () {
```

Comment:
I suppose `EnumerableSet` doesn't care about the role existing. But it felt idiomatic from the `AccessControl` POV to test both cases (role doesn't exist + role exist but index is invalid). But happy to collapse into a single test if needed, please let me know.

Reply:
Seems fine as-is.